drivers/edac/edac_ibecc.c

```diff
@@ -371,7 +386,7 @@ int edac_ibecc_init(const struct device *dev)
 		return -ENODEV;
 	}
 
-	if (!ibecc_enabled(bdf)) {
+	if (ibecc_enabled(bdf) == false) {
```

Comment:
If `ibecc_enabled()` returns `bool` then I don't think you need to do this modification (the bool type is a valid boolean statement in itself)